woocommerce/woocommerce
# Set opt-in/out based on geolocation data and DNT headers

Set’s opt in for tracker based on:

- Geolocation data if in/out of EU
- DNT header. Do not track is respected for all users.

To test:

- go to the setup wizard on an install which has not yet opted in or out (wp-admin/index.php?page=wc-setup)
- you should see the box unchecked if you're in the EU (proxy may affect this!)
- you should see the box checked if you're in the US
- If you enable the donottrack setting in your browser and you're in the US, it will be unchecked

Closes #19284
